docs/source/api/v2/servers_id_deliveryservices.rst

```diff
@@ -239,3 +239,81 @@ Response Structure
 
 
 .. [#tenancy] Only the :term:`Delivery Services` visible to the requesting user's :term:`Tenant` will appear, regardless of their :term:`Role` or the :term:`Delivery Services`' actual 'server assignment' status.
+
+``POST``
+========
+Assign an arbitrary number of delivery services to a single server.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------+----------+---------------------------------------------------------------------------------------------+
+	| Name | Required | Description                                                                                 |
+	+------+----------+---------------------------------------------------------------------------------------------+
+	| id   | Yes      | The integral, unique identifier of the server that you want to assign delivery services to. |
+	+------+----------+---------------------------------------------------------------------------------------------+
+
+.. table:: Request Query Parameters
+
+	+---------+----------+------------------------------------------------------------------------------------------+
+	| Name    | Required | Description                                                                              |
+	+=========+==========+==========================================================================================+
+	| replace | Yes      | Must be a 1, true, 0, or false. Although this query parameter is required, it is unused. |
```

Comment:
Is that true? I think TP may be relying on that.

Reply:
I was confused because the Delivery Services that the server is already assigned to are not returned in `:dsIds:` in the response, regardless of the value of `:replace:`. Corrected in 6d4154ac02.